ci/pod/docker-compose.plugin.yml

```diff
@@ -28,42 +28,26 @@ services:
     networks:
       apisix_net:
 
-
   ## keycloak
   apisix_keycloak:
-    image: sshniro/keycloak-apisix:1.0.0
-    environment:
-      KEYCLOAK_USER: admin
-      KEYCLOAK_PASSWORD: 123456
-    restart: unless-stopped
-    ports:
-      - "8090:8080"
-      - "8443:8443"
-    networks:
-      apisix_net:
-
-  ## keycloak
-  # The keycloak official has two types of docker images:
-  # * legacy WildFly distribution
-  # * new Quarkus based distribution
-  # Here we choose new version, because it's mainstream and
-  # supports kcadm.sh to init the container for test.
-  # The original keycloak service `apisix_keycloak` is
-  # third-party personal customized image and for OIDC test only.
-  # We should unify both containers in future.
-  apisix_keycloak_new:
-    container_name: apisix_keycloak_new
+    container_name: apisix_keycloak
     image: quay.io/keycloak/keycloak:18.0.2
```

Comment:
why not use 21.0 or 21.1?

Reply:
`18.0.2` works for now. If needed, we can upgrade the version in another PR. This PR just delete the 3rd old keycloak, not to upgrade.

Reply:
Do you ask suggestions from sshniro about this PR?

Reply:
Email him before, but no response...

Reply:
@sshniro want to hear from you:)

Reply:
Hi, thanks for this PR, at the time of development I had trouble exporting the configurations/roles/scopes. 
These configs (roles/scopes) should be present in the Keycloak container to test the Authorization flows with our APISIX Keycloak plugin.

Therefore the configs were created in a Keycloak container, and then I built an image out of it.
That's what is present in the `sshniro/keycloak-apisix:1.0.0` container.

This PR resolves these issues with roles and scopes created by the script, instead of inbuilt configurations. 
LGTM!

Reply:
@sshniro Identifying and exporting these configurations one by one is too tedious... But I'm glad it's done.

By the way, the `kcadm.sh` admin tool provided by Keycloak itself is really hard to be used. 😭